src/sounddeviceportaudio.cpp

```diff
@@ -697,14 +690,6 @@ int SoundDevicePortAudio::callbackProcess(const unsigned int framesPerBuffer,
     Q_UNUSED(timeInfo);
     Trace trace("SoundDevicePortAudio::callbackProcess " + getInternalName());
 
-    //qDebug() << "SoundDevicePortAudio::callbackProcess:" << getInternalName();
```

Comment:
Because m_bSetThreadPriority is only a guard for the clock reference device. 
In current master it may happens that the engine thread is not at real time priority, because the headphone soundcard gets the priority first. 

If it turn out that we need real time processing for the second soundcard as well, this should be handled in a separate PR.